ebuild-maintenance/general/text.xml

```diff
@@ -0,0 +1,527 @@
+<?xml version="1.0"?>
+<guide self="ebuild-maintenance/general/">
+<chapter>
+<title>Maintainer's checklist</title>
+
+<section>
+<title>Your responsibilities</title>
+<body>
+
+<p>
+This page aids you in getting familiar with the "everyday" tasks and
+responsibilites of a package maintainer in Gentoo. You may find this page
+useful if you're curious about stepping up as a maintainer, returning from a
+long hiatus, or simply looking for tips.
+
+Not everything listed in here need to be incorporated to your workflow, but
+following these steps should keep the Gentoo repository in a clean working
+state, keep your activity up and ease your tasks as a maintainer.
+</p>
+
+<p>
+Your general responsibilities as a maintainer consist of keeping your packages
+up-to-date with upstream releases, responding to and fixing bugs assigned to
+you, and handling possible stabilization of your packages in time. You also
+need to keep up with the Python, Ruby, PHP, etc. ecosystems in Gentoo.
+</p>
+
+<p>
+<e>Rationale: If your package has existing stable keywords you <b>must</b>
+request stabilization of a newer version within a reasonable timeframe (usually
+at least 30 days, but see the
+<uri link="::keywording/#Moving from ~arch to arch">stabilization section
+</uri>), provided there are no regressions compared to the last stable version.
+This prevents stable users from being stuck on old versions of the package with
+no justification.</e>
+</p>
+
+<p>
+<e>Rationale: It's also important to add the latest e.g. Python versions which
+your package supports as soon as possible to avoid conflicts for users when the
+default Python target in the Gentoo profiles changes.</e> See
+<uri link="::ebuild-maintenance/general/#Daily / Occassional pkgcheck scans"/>for
+a tip to quickly check for a PYTHON_COMPAT update in your packages.
+</p>
+
+<p>
+If you're looking for tips on how to be aware of tasks available to you as a
+maintainer, please see the
+<uri link="::ebuild-maintenance/general/#External tools"/> section below.
+</p>
+
+</body>
+
+<subsection>
+<title>Bugzilla</title>
+<body>
+
+<p>
+<uri link="https://bugs.gentoo.org/">Gentoo's Bugzilla</uri> is still the main
+communication channel to reach you with your package state. When a bug report
+gets assigned to you, the first thing you should attempt to do is to confirm
+and replicate it with the information attached to the bug. You should then
```

Comment:
The [style](https://devmanual.gentoo.org/appendices/contributing/index.html#style-guidelines) guidelines mention third person. I personally find this a bit weird because the technical writing I'm used to allows gratuitous use of 'we', but...

So, we need to change instances of 'you' like this.

Reply:
Yeah, I tried to avoid using "we", "our" and "I" due to this. But I feel like the text can still be addressed to **you**. And there are *several* devmanual pages that read to "you".